tileview/src/main/java/com/moagrius/tileview/plugins/MarkerPlugin.java

```diff
@@ -95,6 +95,23 @@ public void addMarker(View view, int left, int top, float relativeAnchorLeft, fl
     addView(view, layoutParams);
   }
 
+  public void removeMarker(View view) {
+    if (view.getParent() == this) {
+      removeView(view);
+    }
+  }
+
+  public void moveMarker(View view, int left, int top) {
+    MarkerPlugin.LayoutParams lp = (LayoutParams) view.getLayoutParams();
+    if (lp == null) {
+      return;
+    }
+    lp.mLeft = left;
```

Comment:
interesting - it worked for me on my first attempt, let me get a little deeper on it.  will post back.

big question though still stands: 

> Try the first demo ("TileView Demo (Assets)" aka TileViewDemoAssets to see the UI center when scaled smaller than the screen.

Does this fix your FIT/CONTAIN issue?

Reply:
> Does this fix your FIT/CONTAIN issue?

Tested the first demo with assets.
If I zoom out, at some point all tiles disappear. They only appear again if I zoom in enough. I'm sorry this is surely not what you expected.

Reply:
what on earth is happening!  sigh.  ok sounds like i need to get serious about this.  might not be able to bang these out week nights like i thought, and may need the weekend

thanks for taking the time!

Reply:
hey finally a win - yes, you're right about setting x and y instead of mLeft and mTop.  i can still use setLeft and setTop though instead of requestLayout (which is a heavy call to be doing for all the events fired during a pinch)

Reply:
Tbh, this is exactly what I'm thinking :D
I even checked if i pulled the right branch.. but yes it's the good one. 
If when you try again, it's working perfectly for you, don't waste your time and tell me, I will start over, reinstall the app, etc (to be sure).

Reply:
Yeah it is still working perfectly for me (the FIT/CONTAIN bit), but I'll try it on other devices to find the behavior you're describing (vanishing tiles).

Reply:
The emulator just gave me the same result, using the branch issue-512. Well, a good night sleep and I hope every will be fine tomorrow.

Reply:
Thanks for double checking.  I'll hopefully give you a little break and really verify everything is good before asking you to take a look again.  I appreciate all your time and feedback (and the hint on the LayoutParams - I really should have known that, glad you did)